src/app/shared/stackblitz/stackblitz-writer.ts

```diff
@@ -1,6 +1,6 @@
 import {HttpClient} from '@angular/common/http';
 import {Injectable} from '@angular/core';
-import {VERSION} from '@angular/material';
+import {VERSION} from '@angular/material/typings';
```

Comment:
This is an interesting issue that came up. Importing from there is invalid. This will be solved if we move the `VERSION` to `core`, but for now this breaks. cc. @jelbourn

Reply:
This is solved with https://github.com/angular/components/pull/15700